### PR TITLE
PR21

### PR DESCRIPTION
**Pull request by TestGitHub100500**
desc

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -1 +1,2 @@
-print('test')
\ No newline at end of file
+print('test')
+123
```
Comment (OksanaEfanova) [inline test.py:2]: comment1

**Review by OksanaEfanova — APPROVED** [review]

File: test.py (hunk 1 above)
Comment (OksanaEfanova) [reply test.py:2]: comment2

Comment (OksanaEfanova) [reply test.py:2]: comment3

Comment (TestGitHub100500) [reply test.py:2]: comment4

File: test.py [hunk 2]
```diff
@@ -1 +1,2 @@
-print('test')
\ No newline at end of file
+print('test')
```
Comment (OksanaEfanova) [inline test.py:1]: c1

**Review by OksanaEfanova — CHANGES_REQUESTED** [review]

**Comment by TestGitHub100500** [comment]
1

**Comment by OksanaEfanova** [comment]
2

**Review by OksanaEfanova — COMMENTED** [review]
1

**Pull request closed without merging**